vignettes/questionnaires.Rmd

```diff
@@ -70,7 +70,7 @@ dataset_vignette(qs)
 adsl <- tribble(
   ~STUDYID, ~USUBJID, ~SITEID, ~ITTFL,    ~TRTP, ~TRTSDT,                      ~DTHCAUS,
```

Comment:
I'm hesitant to summarize anything by ARM- not sure which section this is associated with, but if it should be referencing ADSL variables, then would suggest using TRTxxP  or TRTxxA (TRT01P/TRT01A?) instead of ARM, since there's a direct relationship between TRTP and TRTxxP.

Reply:
Thanks for the review @nbrucken17 @bundfussr 
Since `TRTP/ARM` is not used anywhere in this vignette, can I remove it?

Reply:
I'm fine with removing it.